plugins/stack-overflow/src/home/StackOverflowQuestions/Content.tsx

```diff
@@ -31,7 +31,7 @@ import {
   StackOverflowQuestion,
   StackOverflowQuestionsContentProps,
 } from '../../types';
-
+import './link-styles.css';
```

Comment:
Instead of importing a `css` file, you can use of [makeStyles](https://v4.mui.com/styles/api/#makestyles-styles-options-hook) to create a class with `width: 100%` and use on `<Link>`, works like a charm.

The main reason that we do this is because `@material-ui` make the `css` classes dynamically, avoiding naming conflicts between the `css` files.